src/renderer/views/preload/PreloadProgressView.tsx

```diff
@@ -160,21 +160,21 @@ const PreloadProgressView = observer(() => {
         break;
       case 0x02:
         console.error("Chain is too low. Automatically relaunch.");
-        ipcRenderer.send("relaunch standalone");
+        ipcRenderer.send("relaunch standalone", {reason: "Tip is low."});
         break;
       case 0x03:
         console.error("Chain's tip is stale. Automatically relaunch.");
-        ipcRenderer.send("relaunch standalone");
+        ipcRenderer.send("relaunch standalone", {reason: "Tip is stale."});
         break;
       case 0x04:
         console.error(
           "Haven't received any messages for some time. Automatically relaunch."
         );
-        ipcRenderer.send("relaunch standalone");
+        ipcRenderer.send("relaunch standalone", {reason: "Haven't received message."});
         break;
       case 0x05:
         console.error("Action Timeout. Automatically relaunch.");
-        ipcRenderer.send("relaunch standalone");
+        ipcRenderer.send("relaunch standalone", {reason: "Action Timeout."});
         break;
```

Comment:
Seems like the fallback case is missing. Is it intended?

Reply:
IDK why `default` fallback is missing. but I'm just do not fix what I don't know.

Reply:
Uh is it okay though? I think we might be losing some tracking information of some crashes with unknown errors.

Reply:
I think that's fine. There is only one case of going to the default where the Launcher get an unexpected error code from Headless, but it doesn't look good for the launcher to handle an undefined error code arbitrarily.

Reply:
Well if that's the case I don't see a problem too. Thanks for an explaination!